test-storybooks/portable-stories-kitchen-sink/react/e2e-tests/component-testing.spec.ts

```diff
@@ -39,12 +39,13 @@ test.describe("component testing", () => {
     await page.evaluate(() => window.sessionStorage.clear());
     await sbPage.waitUntilLoaded();
 
-    await page.evaluate(() => {
-      // @ts-expect-error -- temporary to clear statuses between tests, will be removed when we have a clear button in the UI
-      globalThis.clearStatuses()
-    });
-
-    // Ensure that all features are disabled, as previous tests might have enabled them
+    
+    // Ensure that all test results are removed and features are disabled, as previous tests might have enabled them
+    
+    const clearStatusesButton = page.getByLabel('Clear all statuses');
+    if (await clearStatusesButton.isVisible()) {
```

Comment:
should this be an expectation rather than an if statement?
- clear status has to be visible
- click on it

Reply:
the clear button is only visible if there are actual statuses, so here I'm guarding against previous tests not ending up with any statuses for some reason. I don't think it happens today with the current tests though.